hardware/src/CamRobot/RaspiMount_protoboard.scad

```diff
@@ -1,7 +1,22 @@
 $fa = 1;
 $fs = 0.01;
 
-module cm_half(camera_hole_r, camera_hole_pitch_w, camera_hole_pitch_h, upper_hole_h){
+//args
+// num_camera: number of camera(1 or 2)
+// camera_hole_r: radius of camera hole
+// camera _hole_pitch_w: horizontal pitch of camera holes
+// camera_hole_pitch_h: vertical pitch of camera holes
+// camera_distance: distance between hole of 2 cameras. It is used  only for stereo camera mount
+// upper_hole_h: height of upper camera holes
+module RaspiCameraMount(num_camera=2, camera_hole_r=0.1, camera_hole_pitch_w=2.1, camera_hole_pitch_h=1.25, camera_distance=1,  upper_hole_h=1){
```

Comment:
This module does not support multiple cameras more than three, so please replace num_cameras with a boolean argument as follows:  
```suggestion
module RaspiCameraMount(stereo_camera=false, camera_hole_r=0.1, camera_hole_pitch_w=2.1, camera_hole_pitch_h=1.25, camera_distance=1,  upper_hole_h=1){
```

Reply:
Please fix all modules as well.

Reply:
DONE